wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -348,16 +255,15 @@ def run(
         if launch_project.docker_image:
             if len(containers) > 1:
                 raise LaunchError(
-                    "Multiple container configurations should be specified in a yaml file supplied via job_spec."
+                    "Multiple container configurations should be specified under kubernetes."
                 )
             # dont specify run id if user provided image, could have multiple runs
             containers[0]["image"] = launch_project.docker_image
-            image_uri = launch_project.docker_image
             # TODO: handle secret pulling image from registry
         elif not any(["image" in cont for cont in containers]):
             if len(containers) > 1:
                 raise LaunchError(
-                    "Launch only builds one container at a time. Multiple container configurations should be pre-built and specified in a yaml file supplied via job_spec."
+                    "Launch only builds one container at a time. Multiple container configurations should be pre-built and specified under kubernetes."
```

Comment:
What does "under kubernetes" mean here?

Reply:
under kubernetes here could say "in the job spec" which is what we're trying to have users define now